icons/ambulance.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path
+    d="M7 20a2 2 0 1 0 0-4 2 2 0 0 0 0 4ZM17 20a2 2 0 1 0 0-4 2 2 0 0 0 0 4Z" />
+  <path
+    d="M9 18h5m0 0V7a2 2 0 0 0-2-2H4a2 2 0 0 0-2 2v10a1 1 0 0 0 1 1h2m9 0h1M8 9v4m2-2H6m13 7h2a1 1 0 0 0 1-1v-3.28a1 1 0 0 0-.68-.95l-2-.66a1 1 0 0 1-.51-.4L17.3 9.45a1 1 0 0 0-.84-.45H14" />
```

Comment:
Had to lift up the roof and + by 1px so we don't violate the 2px gap rule.

```suggestion
  <path d="M10 10H6" />
  <path d="M14 18V6a2 2 0 0 0-2-2H4a2 2 0 0 0-2 2v11a1 1 0 0 0 1 1h2" />
  <path d="M19 18h2a1 1 0 0 0 1-1v-3.28a1 1 0 0 0-.68-.95l-2-.66a1 1 0 0 1-.51-.4L17.3 9.45a1 1 0 0 0-.84-.45H14" />
  <path d="M8 8v4" />
  <path d="M9 18h6" />
  <circle cx="17" cy="18" r="2" />
  <circle cx="7" cy="18" r="2" />
```

https://lucide-studio.vercel.app/?value=%3Csvg++xmlns=%22http://www.w3.org/2000/svg%22++width=%2224%22++height=%2224%22++viewBox=%220+0+24+24%22++fill=%22none%22++stroke=%22currentColor%22++stroke-width=%222%22++stroke-linecap=%22round%22++stroke-linejoin=%22round%22%3E++%3Cpath+d=%22M10+10H6%22+/%3E++%3Cpath+d=%22M14+18V6a2+2+0+0+0-2-2H4a2+2+0+0+0-2+2v11a1+1+0+0+0+1+1h2%22+/%3E++%3Cpath+d=%22M19+18h2a1+1+0+0+0+1-1v-3.28a1+1+0+0+0-.68-.95l-2-.66a1+1+0+0+1-.51-.4L17.3+9.45a1+1+0+0+0-.84-.45H14%22+/%3E++%3Cpath+d=%22M8+8v4%22+/%3E++%3Cpath+d=%22M9+18h6%22+/%3E++%3Ccircle+cx=%2217%22+cy=%2218%22+r=%222%22+/%3E++%3Ccircle+cx=%227%22+cy=%2218%22+r=%222%22+/%3E%3C/svg%3E

Reply:
Makes sense. Should I commit your changes or do you want me to update from my original file?

Reply:
You can commit it, maybe that will also fix the preview comment.